librz/include/rz_il/rzil_opcodes.h

```diff
@@ -0,0 +1,492 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RZIL_CORE_THEORY_DEFINITIONS_H
```

Comment:
wrong name, please fix

Reply:
@Heersin please make sure the header guards correspond to the header file names.